api/routes/model.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+var validate = require('express-jsonschema').validate;
+
+
+var ModelSchema = {
+    id: '/ModelSchema',
+    type: 'object',
+    properties: {
+      name: {
+        type: 'string',
+        required: true
+      },
+      sdg: {
+        type: 'array',
+        items:{type: 'integer'},
+        required: true
+      },
+      description: {
+        type: 'string',
+        required: true
+      },
+      stages: {
+        type: 'object',
+        patternProperties: {
+          ".*": {
+            type: 'object',
+            properties: {
+              stakeholders: {
+                type: 'array',
+                items: {type: 'string'}
+              },
+              challenges: {
+                type: 'array',
+                items: {type: 'string'}
+              },
+              insights: {
+                type: 'array',
+                items: {type: 'string'}
+              },
+              description: {
+                type: 'string'
+              }
+            }
+          }
+        }
+      }
+    }
+  };
+
+  router.get('/', function(req, res, next) {
```

Comment:
we should probably add a few more get method stuff

- get specific (by id) - this would be a new route
- get by SDG - can probably put into the params of this route